test/test.js

```diff
@@ -1,66 +1,69 @@
+var ObjectId = require('mongodb').ObjectID;
 
 describe('Server', function () {
 	it('should start the server and initialize it properly', function (done) {
-		server = require('./../server.js');
-		supertest(server.app).get('/').expect('Content-Type', "text/html; charset=utf-8").expect(200, done);
+		require('./../config/app.js').start(function (app) {
+			server.app = app;
+			supertest(app).get('/').expect('Content-Type', "text/html; charset=utf-8").expect(200, done);
+		});
 	});
 });
 
 describe('API get request', function () {
 
 	it('should return a JSON file of announcements', function (done) {
-		request.get('http://localhost:8800/announcement', function (err, res, body) {
+		request.get('http://localhost:8800/API/announcement', function (err, res, body) {
 			var parsed = JSON.parse(body);
 			expect(res.statusCode).to.equal(200);
 			expect('contentType', /json/);
 			expect(parsed).to.be.an('array');
 			if (parsed.length > 0) {
-				expect(parsed[0]).to.have.all.keys('_id', 'title', 'description', 'date', 'poster');
+				expect(parsed[0]).to.include.keys('_id', 'title', 'description', 'created', 'school');
 			}
 			done();
 		});
 
 	});
 
-  it('should return a JSON file of general information',function(done){
-      request.get('http://localhost:8800/generalinfo',function(err, res, body){
-          var parsed = JSON.parse(body);
-          expect(res.statusCode).to.equal(200);
-          expect('content-Type',/html/);
-          expect(parsed).to.be.an('array');
-          if(parsed.length > 0){
-              expect(parsed[0]).to.have.all.keys('_id','title','description', 'date', 'category');
-          }
-          done();
-      });
-  });
+	it('should return a JSON file of general information', function (done) {
+		request.get('http://localhost:8800/API/generalinfo', function (err, res, body) {
+			var parsed = JSON.parse(body);
+			expect(res.statusCode).to.equal(200);
+			expect('content-Type', /html/);
+			expect(parsed).to.be.an('array');
+			if (parsed.length > 0) {
+				expect(parsed[0]).to.include.keys('_id', 'title', 'description', 'created', 'category');
+			}
+			done();
+		});
+	});
 
 	it('should return a JSON file of lecturers', function (done) {
-		request.get('http://localhost:8800/lecturer', function (err, res, body) {
+		request.get('http://localhost:8800/API/lecturer', function (err, res, body) {
 			var parsed = JSON.parse(body);
 			expect(res.statusCode).to.equal(200);
 			expect('contentType', /json/);
 			expect(parsed).to.be.an('array');
 			if (parsed.length > 0) {
-				expect(parsed[0]).to.have.all.keys('_id', 'name', 'description', 'imagepath', 'website');
+				expect(parsed[0]).to.include.keys('_id', 'name', 'description', 'created');
 			}
 			done();
 		});
 	});
 
 	it('should return a JSON file of the forum', function (done) {
-		request.get('http://localhost:8800/forum', function (err, res, body) {
+		request.get('http://localhost:8800/forum/item', function (err, res, body) {
```

Comment:
The forum test have to change again. I will do it after my pull request is merged so we don´t have conflicts.